root/etc/e-smith/templates/etc/nethserver/cockpit.allow/10base

```diff
@@ -0,0 +1,19 @@
+
+#
+# Allow groups to use cockpit 
+#
+root
+{ $admins{'group'} || 'domain admins'}
```

Comment:
@davidep what about if the prop gets several group, in theory it is possible ?

Maybe we should split it ?

Reply:
The `group` prop is singular, so only _one_ group can be set.

https://docs.nethserver.org/en/v7/accounts.html#admin-account

Can I have seen some `split()` on `group` somwhere else? Please fix it!

Reply:
yep in my code inside openssh

Reply:
done, thank you